packages/web-components/fast-foundation/src/checkbox/checkbox.ts

```diff
@@ -87,7 +69,7 @@ export class FASTCheckbox extends FormAssociatedCheckbox {
      * @internal
      */
     public clickHandler = (e: MouseEvent): void => {
-        if (!this.disabled && !this.readOnly) {
+        if (!this.disabled) {
```

Comment:
Seems like we've got the same code for the click handler as for the space key. Should we factor this into a shared method?

Reply:
refactored to a method